### DISCUSSION TITLE
Welcome to bleachbit Discussions!

### DISCUSSION THREAD

**Discussion by az0** [bleachbit/bleachbit#1065] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

How to get started

*  For each new topic of discussion, [create a new discussion thread](https://github.com/bleachbit/bleachbit/discussions/new).
* If you are experiencing a bug, [create an issue](https://github.com/bleachbit/bleachbit/issues/new/choose). 
* If you have a feature request, create a [wishlist issue](https://github.com/bleachbit/wishlist/issues/new).

See also the [old forum](https://www.bleachbit.org/forum)

**Comment by synaeus888** [comment]
I have been using Bleachbit for as long as I have had a home PC (about 4+ years). I usually deploy it in conjunction with CCleaner, which is used after BleachBit. Every day (or night) prior to shutdown, both are used.

**Comment by synaeus888** [comment]
I am preparing to move from XP Pro to Win 7 Pro 64-bit. I am wondering if BleachBit 4.2.0 is the best version for it? Secondy, when setting up BleachBit, are there any options I should not tick for this OS?

**Reply by az0** [reply]
@synaeus888 
Windows 7 was end of life as of January 2020, but BleachBit will probably still work well on it. (It should work much better than Windows XP.) 

You can generally use all the same options on Windows 7 as you did on Windows XP, but as you enable each option, it's smart to read each option and especially pay attention to the pop-up warnings.
